client/src/pages/ProfilePaper.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Fragment } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Paper, Grid, Typography, Button, ButtonGroup } from '@material-ui/core';
+import ProfileSubmission from "./ProfileSubmission";
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    width: "70%",
+    marginLeft: "auto",
+    marginRight: "auto",
+    margin: "5% 0",
```

Comment:
try to avoid using %s! also you may be able to just write `margin: 5% auto` and you won't need the above auto lines

Reply:
will fix that real quick, thanks!